pkg/controller/nodeipam/node_ipam_controller.go

```diff
@@ -28,7 +28,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
remove `v1`

Reply:
I changed this line back (seems to be a default vscode option)